soil_id/utils.py

```diff
@@ -2575,8 +2325,8 @@ def update_esd_data(df):
     Processes the given DataFrame by updating missing ESD data based on component
     groups with the same names, filling missing URLs and ecoclass IDs and names.
     """
-    if "esd_url" not in df.columns:
-        df["esd_url"] = np.nan  # Initialize 'esd_url' as NaN if it does not exist
+    if "edit_url" not in df.columns:
+        df["edit_url"] = np.nan  # Initialize 'edit_url' as NaN if it does not exist
```

Comment:
question: should this be `None` or `""`?

Reply:
Yes, should be: df["edit_url"] = ""
I've updated code